src/components/interactive/yDragBoxLayer.ts

```diff
@@ -0,0 +1,20 @@
+///<reference path="../../reference.ts" />
+
+module Plottable {
+export module Component {
+  export module Interactive {
+    export class YDragBoxLayer extends DragBoxLayer {
+      protected _setBounds(newBounds: Bounds) {
+        super._setBounds({
+          topLeft: { x: 0, y: newBounds.topLeft.y },
+          bottomRight: { x: this.width(), y: newBounds.bottomRight.y }
+        });
+      }
+
+      protected _setResizableClasses(canResize: boolean) {
+        this.classed("y-resizable", canResize);
```

Comment:
I personally like using `super`'s where applicable.  Not sure if applicable here.

Whatever you think here is good.  I don't think I currently have a strong opinion on how to do this well.